src/jbi/models.py

```diff
@@ -57,22 +59,24 @@ class Config:
 
 class Actions(YamlModel):
     """
-    Actions is the overall model for the list of `actions` in the configuration file
+    Actions is the container model for the list of actions in the configuration file
     """
 
-    actions: Mapping[str, Action]
+    __root__: Mapping[str, Action]
+
+    def __len__(self):
+        return len(self.__root__)
+
+    def __getitem__(self, item):
+        return self.__root__[item]
 
     def get(self, tag: Optional[str]) -> Optional[Action]:
         """Lookup actions by whiteboard tag"""
-        return self.actions.get(tag.lower()) if tag else None
-
-    def all(self):
-        """Return mapping of all actions"""
-        return self.actions
+        return self.__root__.get(tag.lower()) if tag else None
```

Comment:
Shouldn't we move this to `__getitem__()` ?

Reply:
Yeah, but one could rely on the other to match the `actions[item.lower()]`  and `actions[None]` behaviour